sphinxcontrib/aafig.py

```diff
@@ -77,7 +78,8 @@ def run(self):
         aafig_options = dict()
         image_attrs = dict()
         own_options_keys = list(self.own_option_spec.keys()) + ['scale']
-        for (k, v) in self.options.items():
+        for (k, v) in tuple(self.options.keys()):
```

Comment:
Did you tried this? I think this won't work, you need to only use one value to iterate.

```suggestion
        for k in tuple(self.options.keys()):
```